src/app/guards/TenantGuard/SsoUserMessage.tsx

```diff
@@ -0,0 +1,41 @@
+import { Stack, Typography } from '@mui/material';
+import FullPageWrapper from 'app/FullPageWrapper';
+import MessageWithLink from 'components/content/MessageWithLink';
+import SingleLineCode from 'components/content/SingleLineCode';
+import AlertBox from 'components/shared/AlertBox';
+import { useIntl } from 'react-intl';
+
+function SsoUserMessage() {
+    const intl = useIntl();
+
+    return (
+        <FullPageWrapper fullWidth={true}>
```

Comment:
Does the boolean need to be passed to this prop explicitly? Does `<FullPageWrapper fullWidth>` not suffice in this instance?